src/index.ts

```diff
@@ -88,7 +88,14 @@ async function runGlobs(options: Options, pluginsDefaultOptions: PluginsOptions,
 
   const ignoreManualFilesNames = options.ignore ? options.ignorePath || [] : [];
   const ignoreManualFilesPaths = ignoreManualFilesNames.map((fileName) => path.resolve(fileName));
-  const ignoreManualFilesContents = await Promise.all(ignoreManualFilesPaths.map((filePath) => fs.readFile(filePath, "utf8")));
+  const ignoreManualFilesContents = await Promise.all(ignoreManualFilesPaths.map(async (filePath) => {
+    try {
+      return await fs.readFile(filePath, "utf8");
+    } catch {
+      // Treat a missing file as if it was empty (no ignores)
+      return "";
```

Comment:
im not a huge fan of creating these empty strings, but if we don't, `ignoreManual` will end up empty and we'll apply default `gitignore` etc

prettier doesn't do that, it treats a non-existent file as if it existed (i.e. it turns off the defaults)

Reply:
Arguably the behavior in v4 is better, like you point the CLI to something that doesn't exist and we should just silently fail? 🤔 Pretty weird, but ok we can align with that.